examples/rocker_switch_example.cpp

```diff
@@ -0,0 +1,31 @@
+#include "rocker_switch.h"
+
+#include <Arduino.h>
+
+// Pin definitions
+#define ROCKER_UP 0
+#define ROCKER_DOWN 5
+
+RockerSwitch rocker(ROCKER_UP, ROCKER_DOWN);
+
+void setup()
+{
+  Serial.begin(9600);
+  Serial.println("Started Serial communication");
```

Comment:
This print statement is only called for the rocker switch, not for the other examples. Probably be best to remove it.